File: java/src/main/java/io/fivexer/sdk/model/WorkerBreakEnded.java
package io.fivexer.sdk.model;

/**
 * The result of ending a break.
 *
 * <p>Populated by Gson via field reflection and exposed read-only; callers receive instances
 * from the client and never construct them.
 */
public class WorkerBreakEnded {
    private String workerId;
    private boolean onBreak;
    private String endedAt;

    public String getWorkerId() { return workerId; }
    public boolean isOnBreak() { return onBreak; }
    public String getEndedAt() { return endedAt; }
}

File: php/src/Model/WorkerBreakEnded.php
<?php

declare(strict_types=1);

namespace Fivexer\SDK\Model;

/**
 * The result of ending a break.
 *
 * Response model: built from a decoded JSON array and read-only thereafter.
 */
final class WorkerBreakEnded
{
    public function __construct(
        public readonly string $workerId,
        public readonly bool $onBreak,
        public readonly string $endedAt,
    ) {
    }

    /**
     * @param array<string, mixed> $data
     * @internal
     */
    public static function fromArray(array $data): self
    {
        return new self(
            workerId: (string) ($data['workerId'] ?? ''),
            onBreak: (bool) ($data['onBreak'] ?? false),
            endedAt: (string) ($data['endedAt'] ?? ''),
        );
    }
}
